std/exception.d

```diff
@@ -2145,3 +2145,58 @@ pure nothrow @safe unittest
 
     auto infSlice = infinite[0 .. $]; // this would throw otherwise
 }
+
+
+/**
+ * Convenience mixin for trivially sub-classing exceptions
+ *
+ * Even trivially sub-classing an exception involves writing boilerplate code
+ * for the constructor to correctly pass in the source file and line number the
+ * exception was thrown from. This mixin makes it easier.
```

Comment:
Not sure I get you. Did you mean like if somebody subclassed an exception but which takes the `file` and `line` arguments in the wrong order? But then if someone used it with `enforce` it would result in a compile error, since `string` cannot be interchanged with `size_t`, no?